client/components/About/About.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+
+import styles from './About.scss'
+
+class About extends Component {
+  render () {
+    return (
+      <section>
+        <h1 className={styles.contactTitle}>Contact</h1>
```

Comment:
Don't worry about styling `h1`s right now. We have a global `h1` style for them to keep them the same size and margins across the app. I know the font-size is pretty small but we're working with it for now!